### PR TITLE
empty game fragment

### PR DESCRIPTION
**Pull request by ronigrr**

### CODE REVIEW

File: app/src/main/java/com/example/makeiteven2/fragments/FragmentLevelsScreen.kt [hunk 1]
```diff
@@ -47,13 +47,15 @@ class FragmentLevelsScreen : Fragment() {
...
     }
+
     override fun onAttach(context: Context) {
-        if (context is IFragmentLevelsScreenCallback) {
-            mCallBack = context
-        }
-        else{
-            throw ClassCastException(context.toString() + "must implement OnLevelClicked")
-        }
+//        if (context is IFragmentLevelsScreenCallback) {
```
Comment (ilya3a) [inline app/src/main/java/com/example/makeiteven2/fragments/FragmentLevelsScreen.kt]: why in comment?

File: app/src/main/java/com/example/makeiteven2/fragments/FragmentLevelsScreen.kt [hunk 2]
```diff
@@ -15,15 +15,15 @@ import kotlinx.android.synthetic.main.fragment_levels.view.*
 
 class FragmentLevelsScreen : Fragment() {
 
-    private lateinit var mCallBack : IFragmentLevelsScreenCallback
+    //private lateinit var mCallBack : IFragmentLevelsScreenCallback
```
Comment (ilya3a) [inline app/src/main/java/com/example/makeiteven2/fragments/FragmentLevelsScreen.kt:18]: ??

File: app/src/main/java/com/example/makeiteven2/adapters/LevelsAdapter.kt [hunk 3]
```diff
@@ -63,7 +80,7 @@ class LevelsAdapter(private var mLevelsList : List<Level> = ArrayList(), private
     }
 
     override fun onBindViewHolder(holder: ViewHolder, position: Int) {
-        mLevelsList[position].let { list ->holder.bind(list,mCurrentStage) }
+        mLevelsList[position].let { list ->holder.bind(list,mCurrentStage,mCallBack) }
```
Comment (ilya3a) [inline app/src/main/java/com/example/makeiteven2/adapters/LevelsAdapter.kt:83]: why you bind the list? and not the element in the list?
ok got it the parameter name is list should be level

File: app/src/main/java/com/example/makeiteven2/adapters/LevelsAdapter.kt [hunk 4]
```diff
@@ -10,15 +10,31 @@ import androidx.recyclerview.widget.RecyclerView
...
+        }
+    }
+
+    interface ILevelsAdapter{
+        fun levelsAdapterItemClicked(levelNumber : Int)
+    }
+
     class ViewHolder(private val view : View,private val context: Context) : RecyclerView.ViewHolder(view){
 
         @SuppressLint("ClickableViewAccessibility")
```
Comment (ilya3a) [inline app/src/main/java/com/example/makeiteven2/adapters/LevelsAdapter.kt:36]: what does it mean?

**Review by ilya3a — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/makeiteven2/fragments/FragmentLevelsScreen.kt (hunk 1 above)
Comment (ronigrr) [reply app/src/main/java/com/example/makeiteven2/fragments/FragmentLevelsScreen.kt]: not using it at the moment , deleted

File: app/src/main/java/com/example/makeiteven2/fragments/FragmentLevelsScreen.kt (hunk 2 above)
Comment (ronigrr) [reply app/src/main/java/com/example/makeiteven2/fragments/FragmentLevelsScreen.kt:18]: not using callback at the moment

File: app/src/main/java/com/example/makeiteven2/adapters/LevelsAdapter.kt (hunk 4 above)
Comment (ronigrr) [reply app/src/main/java/com/example/makeiteven2/adapters/LevelsAdapter.kt:36]: what do you mean?

Comment (ilya3a) [reply app/src/main/java/com/example/makeiteven2/adapters/LevelsAdapter.kt:36]: @SuppressLint("ClickableViewAccessibility") ??

Comment (ronigrr) [reply app/src/main/java/com/example/makeiteven2/adapters/LevelsAdapter.kt:36]: oo it is to suppress warnings

**Review by ilya3a — APPROVED** [review]

**Pull request merged**